### PR TITLE
improve(insured-bridge): Adjustments for OVM 2.0 changes

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

This PR introduces no logical changes, only changes to constant values.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/contracts/external/ovm/OVM_CrossDomainEnabled.sol [hunk 1]
```diff
@@ -1,10 +1,9 @@
-// Copied from: https://github.com/ethereum-optimism/optimism/blob/5fc728da7381eda6b6f858c091636c947186a3ce/packages/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol
-
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
 /**
  * @title iOVM_CrossDomainMessenger
+ * @notice Copied directly from https://github.com/ethereum-optimism/optimism/blob/294246d65e650f497527de398f51b2f6d11f602f/packages/contracts/contracts/libraries/bridge/ICrossDomainMessenger.sol
```
Comment (nicholaspai) [inline packages/core/contracts/external/ovm/OVM_CrossDomainEnabled.sol:6]: Updated links to OVM 2.0 code

File: packages/core/contracts/insured-bridge/ovm/OVM_BridgeDepositBox.sol [hunk 2]
```diff
@@ -7,21 +7,20 @@ import "../../external/ovm/OVM_CrossDomainEnabled.sol";
...
+ * @dev Copied from https://github.com/ethereum-optimism/optimism/blob/b6fa9e4085da581fb47993b9c895fa20a631897d/packages/contracts/contracts/libraries/constants/Lib_PredeployAddresses.sol
  */
 library Lib_PredeployAddresses {
     address internal constant L2_TO_L1_MESSAGE_PASSER = 0x4200000000000000000000000000000000000000;
     address internal constant L1_MESSAGE_SENDER = 0x4200000000000000000000000000000000000001;
     address internal constant DEPLOYER_WHITELIST = 0x4200000000000000000000000000000000000002;
-    address internal constant ECDSA_CONTRACT_ACCOUNT = 0x4200000000000000000000000000000000000003;
-    address internal constant SEQUENCER_ENTRYPOINT = 0x4200000000000000000000000000000000000005;
-    address internal constant OVM_ETH = 0x4200000000000000000000000000000000000006;
+    address payable internal constant OVM_ETH = payable(0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000);
```
Comment (nicholaspai) [inline packages/core/contracts/insured-bridge/ovm/OVM_BridgeDepositBox.sol:16]: Looks like these predeploy addresses changed slightly in OVM 2.0

File: packages/core/contracts/insured-bridge/ovm/OVM_CrossDomainEnabled.sol [hunk 3]
```diff
@@ -1,84 +0,0 @@
-// Copied directly from https://github.com/ethereum-optimism/optimism/blob/7ffb83509c589bd35c2e5f9cb2c6ccbd5f346175/packages/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol
```
Comment (nicholaspai) [inline packages/core/contracts/insured-bridge/ovm/OVM_CrossDomainEnabled.sol:1]: This was redundant with the OVM_CrossDomainEnabled contract in `external`, so I remove this one in favor of the `external` one.

**Review by mrice32 — COMMENTED** [review]
Do we need these changes checked or audited by OZ?

**Comment by nicholaspai** [comment]
> Do we need these changes checked or audited by OZ?

Theroetically yes because there are contract changes but if you review the contract changes there are no logic changes. Files are moved around, and some constant addresses are changed.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**